### PR TITLE
chore: small refactor + mock delay in test, reducing 50s the test duration

### PR DESCRIPTION
**Pull request by pedrotambo**

### CODE REVIEW

File: content/src/logic/cluster-helpers.ts [hunk 1]
```diff
@@ -59,11 +59,8 @@ export async function determineCatalystIdentity(
...
-      attempts--
-      if (attempts > 0) {
-        logger.debug(`Retrying`, { attempts })
-        await delay(IDENTITY_DETERMINATION_ATTEMPTS_INTERVAL)
-      }
+      await delay(attempIntervalMs)
+      attempts++
     }
   } catch (error) {
     logger.error(`Failed to detect my own identity \n${error}`)
```
Comment (agusaldasoro) [inline content/src/logic/cluster-helpers.ts:66]:
```suggestion
    logger.error(`Failed to detect my own identity after ${attempts} attempts \n${error}`)
```

File: content/test/unit/service/synchronization/ContentCluster.spec.ts [hunk 2]
```diff
@@ -7,11 +7,17 @@ import { createFetchComponent } from '../../../../src/ports/fetcher'
 import { ChallengeSupervisor, IChallengeSupervisor } from '../../../../src/service/synchronization/ChallengeSupervisor'
 import { ContentCluster } from '../../../../src/service/synchronization/ContentCluster'
 import { MockedDAOClient } from '../../../helpers/service/synchronization/clients/MockedDAOClient'
+jest.mock('@dcl/catalyst-node-commons', () => ({
+  ...jest.requireActual('@dcl/catalyst-node-commons'),
+  delay: jest.fn()
+}));
+console.log(delay)
```
Comment (agusaldasoro) [inline content/test/unit/service/synchronization/ContentCluster.spec.ts:14]: is this log necessary?

**Review by agusaldasoro — APPROVED** [review]
👏

File: content/test/unit/service/synchronization/ContentCluster.spec.ts (hunk 2 above)
Comment (pedrotambo) [reply content/test/unit/service/synchronization/ContentCluster.spec.ts:14]: Nope haha

**Pull request merged**